src/fusor/translator.py

```diff
@@ -187,40 +182,18 @@ def _get_genomic_ac(self, chrom: str, build: Assembly) -> str:
 
     async def from_jaffa(
         self,
-        fusion_genes: str,
-        chrom1: str,
-        base1: int,
-        chrom2: str,
-        base2: int,
-        rearrangement: bool,
-        classification: str,
-        inframe: bool,
-        spanning_reads: int,
-        spanning_pairs: int,
+        jaffa: JAFFA,
         coordinate_type: CoordinateType,
         rb: Assembly,
     ) -> AssayedFusion | None:
         """Parse JAFFA fusion output to create AssayedFusion object
 
-        :param fusion_genes: A string containing the two fusion partners
-        :param chrom1: The chromosome indicated in the chrom1 column
-        :param base1: The genomic position indicated in the base1 column
-        :param chrom2: The chromosome indicated in the chrom2 column
-        :param base2: The genomic position indicated in the base2 column
-        :param rearrangement: A boolean indicating if a rearrangement occured
-        :param classification: The classification associated with the called fusion
-        :param inframe: A boolean indicating if the fusion occurred in-frame
-        :param spanning_reads: The number of deteced reads that span the junction
-            bewtween the two transcript. Although described as spanning reads, this
-            aligns with our defintion of split reads (i.e. reads that have sequence
-            belonging to the fusion partners)
-        :param spanning_pairs: The number of detected reads that align entirely on
-            either side of the breakpoint
+        :param JAFFA: A JAFFA object
```

Comment:
This is a general comment about description for the callers. Consider changing these to be like "Output from JAFFA caller", since the function signature indicates its expecting a JAFFA object